README.md

```diff
@@ -1 +1,172 @@
+* Use `process.cwd()` when working with a directory unrelated to your code. For example, if you are creating a command-line tool which generates a boilerplate code, you'll want to be able to use the tool in any directory to create new directories and files relative to the location in which you launch the tool and not in the location of the tool. Typically developers use `process.cwd()` for command-line tools.
+
+Let's cover the very close relative to `cwd`, the `process.env.PWD`.
+
+## Understanding process.env.PWD
+
+The `process.env.PWD` is similar to the `process.cwd()` in the value of the absolute path. It will return the value of the directory from which you launched the Node process. PWD stands for Print Working Directory. For example, with `node node-modules-path/node customers`, the `PWD` variable will be ` /Users/azat/Documents/Code/learn-co` the same as the value of `process.cwd().
+
+
+However, there are a few differences between `PWD` and `cwd()`. `PWD` is an environment variable and not a function. `PWD` is a call to the command on Unix-like systems:
+
+```
+pwd
+```
+
+You can also print the environment variable from the command line with `echo`:
+
+```
+echo $PWD
+```
+
+Another difference is that `PWD` can be changed with an assignment, just like any environment variable, while the current working directory can be changed with `process.chdir()`. Changing `PWD` won't affect the value of `cwd()`.
+
+Also, the `PWD` env var is a POSIX (Unix, Linux, Mac OS X) variable which mean it won't work on Windows. The `process.cwd()` will work on Windows.
+
+The `PWD` might be useful on POSIX systems (Unix, Linux, Mac OS X) to get the original value of the current working directory of the process.
+
+
+## Importing a Folder
+
+Sometime you need to import a few related files. Let's say you are working with accounts and you have modules to add, delete, update and read accounts:
+
+```
+add-accounts.js
+read-accounts.js
+update-accounts.js
+delete-accounts.js
+```
+
+In your main file you can import them one by one with `require(FILEPATH)`:
+
+```js
+var addAccounts = require('add-accounts.js')
+var readAccounts = require('read-accounts.js')
+var updateAccounts = require('update-accounts.js')
+var deleteAccounts = require('delete-accounts.js')
+addAccounts()
+readAccounts()
+...
+```
+
+Now let's imagine you have two more files in which you need to use the account functionality. You would repeat the four lines in those files. That's okay until it's not okay. What if instead of requiring four files, the new requirement is to add 10 more files. You need to add 10 more lines to each of those three "main" files which import the modules. Or maybe you want to initialize the accounts with some constructor code before performing the action like add, create, delete or update.
```

Comment:
I think this section is really important and generally well done, but I'm not sure what it has to do with the discussion above. Can we tie these together better?